input.cpp

```diff
@@ -136,7 +141,7 @@ bool MoonlightInstance::HandleInputEvent(const pp::InputEvent& event) {
     switch (event.GetType()) {
         case PP_INPUTEVENT_TYPE_MOUSEDOWN: {
             // Lock the mouse cursor when the user clicks on the stream
-            if (!m_MouseLocked) {
+            if (m_MouseLockingFeatureEnabled && !m_MouseLocked) {
```

Comment:
We should still provide toggleable input capture behavior like what Moonlight Qt does when in remote desktop mouse mode, otherwise it is impossible to uncapture input to use the shortcut keys required to exit fullscreen.

When mouse locking is disabled and the user presses Ctrl+Alt+Shift, we should:
- Toggle cursor visibility
- Toggle ignoring keyboard and mouse input

Reply:
Thanks for your feedback. I'm happy to implement the change, but personally I've not used the Ctrl+Alt+Shift shortcut as both of my chromebooks (and so I assumed _all_ chromebooks) feature function keys to toggle the fullscreen and window-switching functionality, which I found a more obvious way to navigate.